pilz_testutils/include/pilz_testutils/joint_state_publisher_mock.h

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (c) 2020 Pilz GmbH & Co. KG
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef PILZ_TESTUTILS_JOINT_STATE_PUBLISHER_MOCK_H
+#define PILZ_TESTUTILS_JOINT_STATE_PUBLISHER_MOCK_H
+
+#include <atomic>
+#include <mutex>
+#include <string>
+#include <thread>
+#include <vector>
+
+#include <ros/ros.h>
+
+#include <sensor_msgs/JointState.h>
+
+namespace pilz_testutils
+{
+/**
+ * @brief Mocks the joint_states interface. Can simulate robot movement by changing the position of the first joint in
+ * \c controller_joint_names.
+ */
+class JointStatePublisherMock
+{
+  typedef sensor_msgs::JointState JointState;
+  typedef sensor_msgs::JointStateConstPtr JointStateConstPtr;
+
+public:
+  /**
+   * @brief Constructor.
+   * @param ns Namespace in which the joint states are to be published.
```

Comment:
No param. Rather remove this comment all together if it does not state something which is not already obvious.